fortuna/src/Login.js

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import ReactDOM from 'react-dom';
+import MarketPlaceDisplay from './MarketPlace.js';
+const styleForRow = {
+    'margin-top':'20px',
+  };
+  
+  //This is the Login Page Display
+  class LoginPageDisplay extends React.Component {
+    handleLoginClick = () => {
```

Comment:
This looks like it's a global variable being declared by accident in a method. Use const instead to avoid shooting yourself in the foot later by accident :)

Reply:
Const brings an error that it can't be done in javascript